.github/workflows/java-eks-retry.yml

```
@@ -34,6 +37,7 @@ jobs:
       aws-region: ${{ inputs.aws-region }}
       test-cluster-name: ${{ inputs.test-cluster-name }}
       caller-workflow-name: ${{ inputs.caller-workflow-name }}
+      java-version: '11'
```

Use the input here and below
```suggestion
      java-version: ${{ inputs.java-version }}
```